### DISCUSSION TITLE
Transparency shader showing overlaps - avoidable?

### DISCUSSION THREAD

**Discussion by phreak** [google/model-viewer#4317] [Q&A]
Hi all, when working with transparency shaders, the faces overlap unbeautifully.

Is there a know solution to create just a transparent object without showing the "inner life" (see the screenshot).

<img width="859" alt="Bildschirm­foto 2023-06-12 um 20 53 12" src="https://github.com/google/model-viewer/assets/368430/a571ef53-d8fb-4235-9bba-3c725b9d1929">

**Comment by elalish** [comment]
No, how would we know what parts you want to show and what parts you don't? You'll have to edit your model.
